lib/pages/login_page.dart

```diff
@@ -445,7 +447,7 @@ class _LoginPageState extends State<LoginPage>
 
   Widget _buildSignUp(BuildContext context) {
     return Container(
-      padding: EdgeInsets.only(top: 23.0),
+      padding: EdgeInsets.only(top: 0.0),
```

Comment:
if no padding is need then remove this padding field rather that assigning its value to 0

Reply:
okay @hiashutoshsingh

Reply:
@hiashutoshsingh  have removed the padding field whose value is assigned to 0